src/utils.ts

```diff
@@ -83,3 +83,12 @@ export const getWorkspacePath = (logger: winston.Logger): string | void => {
     return;
 };
 
+export const runVersionCommand = (logger: winston.Logger, checkovPath: string, workingDir: string | undefined): void => {
```

Comment:
Love it!
Maybe it'll even be useful to add it as a command (to enable the user to initiate it at will)

Reply:
Yeah, I think I want to add a new command for this and also to open the log file, but I will do that as a new change.

Reply:
Actually, I will do it in this PR so that I can reference the command from the settings page.